recipes/vc-feature/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: vc-feature
+  version: {{ version }}
+
+build:
+  number: 0
+  track_features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
```

Comment:
Yeah its tough. On the one hand you want to make sure they are kicked off correctly on the Python version, but you also want the flexibility of just selecting the right compiler depending on whatever features are activated. I think this is kind of cycling back around to all of the discussions about how best to sort out features/different build matrices. For now its probably easiest to ensure that Python is there - then when we sort everything out we can pull Python and add in the proper matrices. Although the runtimes may also be a better idea but @msarahan probably knows better than me if that is a good idea.